hllplus/sparse.go

```diff
@@ -60,6 +60,8 @@ func (s *sparseState) Add(hash uint64) {
 
 // Linear counting over the number of empty sparse buckets.
 func (s *sparseState) Estimate() int64 {
+	s.Flush()
```

Comment:
@dim please, note this addition. Google implementation does this (if I'm reading Java correctly) on each estimate; tests were not failing without this, but still - I decided that explicit flush is needed, as buffer count is not really done here.

Though, we could do a simple:

```go
float64(s.data.Count() + s.buffer.Len())
```

or so, that should be more efficient and still precise?

Reply:
Oh, actually that wouldn't be precise, as buffer may have elements, that are already stored in `s.data`, so Flush is unavoidable for precise estimate.

Reply:
yep, you need to flush, as buffer and data could have dupes